src/components/application_manager/test/application_impl_test.cc

```diff
@@ -671,6 +674,8 @@ TEST_F(ApplicationImplTest, UpdateHash_AppMngrNotSuspended) {
 }
 
 TEST_F(ApplicationImplTest, UpdateHash_AppMngrSuspended) {
+  ON_CALL(mock_application_manager_settings_, hash_string_size())
+      .WillByDefault(Return(0u));
```

Comment:
@ValeriiMalkov better to make a constant from 0u, as an `empty_hash_string_size`